src/status_im/contexts/wallet/common/screen_base/create_or_edit_account/view.cljs

```diff
@@ -30,7 +31,9 @@
       {:customization-color account-color
        :container-style     (style/gradient-cover-container margin-top)}]
      (into
-      [:<>
+      [rn/scroll-view
```

Comment:
I think we should use the floating_button_page component on this screen instead, it's the same behaviour required 👍

Reply:
changed it to using floating-button-page. Had to add an extra prop so @ulisesmac could you also have a look, since u also worked on it?